Translations/Portuguese/Contribution Guide.md

```diff
@@ -1,2 +1,63 @@
 This is the link  to the Contribution Guide to be translated.
 [Contribution Guide](https://github.com/chaoss/chaoss-slack-bot/wiki/Contribution-Guide)
+
+# Guia de contribuição
+
+Adoramos receber pull requests de todos! Seguimos o fluxo de trabalho padrão do Git de fork -> change -> pull request -> merge -> update fork -> change ... (repetir para sempre). Se você é novo no código aberto, recomendamos o excelente guia do GitHub sobre "[Como contribuir para o código aberto](https://opensource.guide/how-to-contribute/)". Além disso, sinta-se à vontade para entrar em contato com qualquer um dos mantenedores ou outros membros da comunidade se estiver com dificuldades; estamos aqui para ajudá-lo a aprender!
+
+## Abrindo um problema
+
+Se você estiver enfrentando problemas com o CHAOSS Slack Bot ou tiver alguma dúvida que gostaria de ajuda para responder, sinta-se à vontade para abrir um [issue](https://github.com/chaoss/chaoss-slack-bot/issues). Para nos ajudar a evitar duplicatas, pedimos que você pesquise brevemente o seu problema ou dúvida em nossas [issues](https://github.com/chaoss/chaoss-slack-bot/issues) antes de abrir uma nova. Observe que se você abrir um relatório de bug, não poderemos ajudá-lo até que você nos forneça todas as informações relevantes. Respeitosamente, não temos tempo para tentar recriar um erro fornecido com o mínimo ou nenhum contexto, portanto, ao fornecer essas informações, você está nos ajudando a ajudá-lo! Forneça descrições da melhor maneira possível e inclua capturas de tela e erroros de logs, se aplicável.
```

Comment:
It should be nice if we have one line per sentence. 
@Busayo-ojo do you have any thoughts?